### DISCUSSION TITLE
Installation Failed in Laravel 11.31 with php 8.3

### DISCUSSION THREAD

**Discussion by jarwonozt** [botman/botman#1417] [Q&A]
I tried to install Botman on Laravel 11.32 with PHP version 8.3, but an error message appeared during the installation process, as shown in the image below. Is there a solution or do I have to wait for the latest update from botman, so that the packages are in sync.

![image](https://github.com/user-attachments/assets/bd389eb9-be32-4617-a1a6-1aef21982191)

**Comment by filippotoso** [comment]
Try now.
